guides/release/routing/defining-your-routes.md

```diff
@@ -76,7 +76,7 @@ Router.map(function() {
 
 will still by default use the `blog-post.js` route handler and the
 `blog-post.hbs` template, but will be referred to as `blog_post` in any
-`{{link-to}}` helpers.
+`<LinkTo>` helpers.
```

Comment:
Shouldn’t the short form be `<LinkTo />`?

Reply:
We used `{{some-name}}` to refer to components and helpers regardless of the forms they supported, so I was thinking we do the same here.